lld/ELF/Config.h

```diff
@@ -547,6 +547,8 @@ struct Ctx {
   LinkerScript *script;
   std::unique_ptr<TargetInfo> target;
 
+  ErrorHandler *e;
```

Comment:
I think it would be worth using a full name as it isn't used frequently enough to benefit from a short abbreviation.

`ctx.e` could be almost anything. Perhaps ctx.errHandler.

Reply:
+1